tools/gulptasks/test-docs.js

```diff
@@ -23,6 +24,7 @@ async function checkDocsConsistency() {
         );
     }
     tsFiles.forEach(file => {
+        file = FSLib.path(file, true);
```

Comment:
This should not be necessary, and in case it is, the posix flag should not be set, so that posix paths get converted to Windows style.

Reply:
This is cherrypicked from #21487